lightly_studio/src/lightly_studio/resolvers/video_resolver/get_all_by_dataset_id.py

```diff
@@ -0,0 +1,60 @@
+"""Implementation of get_all_by_dataset_id function for videos."""
+
+from __future__ import annotations
+
+from collections.abc import Sequence
+from uuid import UUID
+
+from pydantic import BaseModel
+from sqlmodel import Session, col, func, select
+
+from lightly_studio.api.routes.api.validators import Paginated
+from lightly_studio.models.sample import SampleTable
+from lightly_studio.models.video import VideoTable
+
+
+class VideosWithCount(BaseModel):
+    """Result of getting all samples."""
+
+    samples: Sequence[VideoTable]
```

Comment:
Not sure if this should be called `videos` but let's keep samples for now - it will be easier to factor the common sample functionality out later.